sdk/core/core-http/package.json

```diff
@@ -56,6 +56,9 @@
     "./dist-esm/src/util/inspect.js": "./dist-esm/src/util/inspect.browser.js",
     "./dist-esm/src/util/url.js": "./dist-esm/src/util/url.browser.js"
   },
+  "react-native": {
```

Comment:
Since react native support will be only part of core-v2, should we add the mapping here?